Automation/Python/Pytest tips & tricks.md

```diff
@@ -0,0 +1,91 @@
+## What is pytest?
+
+Pytest is a testing framework for Python.
+For more info, check the [documentation](https://pytest.org).
+
+
+### Conftest
+
+`conftest.py` is a file in which you can set up your test configuration. 
+The file should primarily consist of fixtures which are automatically detected and executed.
+
+You can have multiple `conftest` files in your project. 
+For example, you can have one `conftest` in the project root where you want to set up logic for the entire project. This one might be useful for the overall configuration, like setting up driver(s).
+
+And then you can add additional `conftest` file into your tests directory.
```

Comment:
Should it be "add **an** additional `conftest` file" or "add additional `conftest` file**s**"? 😄

Reply:
Part one of the paragraph talks about adding a (one) conftest file as root.
The second part says you can override it (or add additional logic just for tests) by adding one more.
Only one conftest can be in each directory. So `file`

Reply:
Added missing article

Reply:
Perfect 🚀